### PR TITLE
update contributing.rst

### PR DESCRIPTION
**Pull request by ringabout**
Contributing guide a bit outdated, so I fix it

https://github.com/nim-lang/Nim/blob/da4215af6a5046662db736ce2d5500ce0f38fe1b/doc/contributing.rst

Now most of these libraries
algorithms
asyncdispatch
asyncfile
asyncnet
asyncstreams
base64
bitops
cgi
colors
complex
cookies
coro
cstrutils
distros
encodings
endians
hashes
htmlgen
htmlparser
httpclient
httpcore
json
......

use third-person singular

### CODE REVIEW

File: doc/contributing.rst [hunk 1]
```diff
@@ -255,27 +255,27 @@ first appearance of the proc.
 .. code-block:: nim
 
   proc hello*(): string
-    ## Put documentation here
+    ## Puts documentation here
   proc nothing() = discard
   proc hello*(): string =
-    ## ignore this
+    ## Ignores this
```
Comment (metagn) [inline doc/contributing.rst]: I think this is supposed to mean "this gets ignored"?

File: doc/contributing.rst [hunk 2]
```diff
@@ -255,27 +255,27 @@ first appearance of the proc.
 .. code-block:: nim
 
   proc hello*(): string
-    ## Put documentation here
+    ## Puts documentation here
```
Comment (ringabout) [inline doc/contributing.rst]:
```suggestion
    ## Put documentation here
```

File: doc/contributing.rst (hunk 1 above)
Comment (ringabout) [inline doc/contributing.rst]:
```suggestion
    ## Ignore this
```

File: doc/contributing.rst [hunk 3]
```diff
@@ -258,24 +258,24 @@ first appearance of the proc.
     ## Put documentation here
   proc nothing() = discard
   proc hello*(): string =
-    ## ignore this
+    ## Ignore this
```
Comment (ringabout) [inline doc/contributing.rst]:
```suggestion
    ## ignore this
```

**Pull request merged**